instruments/thorlabs/tc200.py

```diff
@@ -198,7 +198,11 @@ def temperature_set(self):
         :rtype: `~pint.Quantity`
         """
         response = (
-            self.query("tset?").replace(" C", "").replace(" F", "").replace(" K", "")
+            self.query("tset?")
+            .replace(" Celsius", "")
+            .replace(" C", "")
+            .replace(" F", "")
+            .replace(" K", "")
         )
         return u.Quantity(float(response), u.degC)
```

Comment:
Should we update this to parse the units instead of just throwing them away?

Reply:
According to the [manual)(https://www.thorlabs.com/thorcat/12500/12597-D02.pdf), section 6.3.2:

> All temperature inputs and read backs are in °C only, regardless of what the display units are for the front panel LCD of the TC200. 

I assume it would never return other units than 'C', but then again... didn't except 'Celsius' either.

Reply:
😂 fair enough! haha